std/special/compiler_rt/floatunditf.zig

```diff
@@ -0,0 +1,28 @@
+const builtin = @import("builtin");
+const is_test = builtin.is_test;
+const std = @import("../../index.zig");
+
+pub extern fn __floatunditf(a: u128) f128 {
+    @setRuntimeSafety(is_test);
+
+    if (a == 0) {
+        return 0;
+    }
+
+    const mantissa_bits = std.math.floatMantissaBits(f128);
+    const exponent_bits = std.math.floatExponentBits(f128);
+    const exponent_bias = (1 << (exponent_bits - 1)) - 1;
```

Comment:
Is this missing a right shift? This looks like the max exponent, not the bias.

floatunsitf.zig, line 14: same question.

Reply:
This is correct since we halve the range. For example, a double with exponent bit count of 11 gives `(1 << (11 - 1)) - 1 = 1023`. Max exponent is `(1 << 11) - 1`.